setup.py

```diff
@@ -1,13 +1,14 @@
 # Standard Library
 import os
 import sys
+from datetime import date
 
 # Third Party
 import setuptools
 
-exec(open("smdebug/_version.py").read())
+# First Party
+import smdebug
```

Comment:
We can't import smdebug in setup.py - this is where we build the package in the first place. Why are we switching this around?

Reply:
i can switch it back, but this works too. i have checked.

Reply:
Interesting. As long as you've tested it in a fresh environment where smdebug isn't already installed - just never seen it done this way before.